modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/core/call/CallOutput.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.ignite.cli.core.call;
+
+/**
+ * Output of {@link Call}.
+ * @param <T> type of the body.
+ */
+public interface CallOutput<T> {
+    /**
+     * @return Body of the call. Can be {@link String} or any other type.
+     */
+    T body();
+
+    /**
+     * @return Status of the call execution.
+     */
+    CallOutputStatus status();
+
+    /**
+     * @return True if status is {@link CallOutputStatus#ERROR}.
```

Comment:
Can we assert that using the default method? In the DefaultCallOutput this method is implemented differently.